src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,20 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Car car = carService.get(3L);
+        Driver driver = driverService.get(2L);
+        carService.addDriverToCar(driver, car);
+        carService.getAll().forEach(System.out::println);
```

Comment:
It's not enough, let's create here drivers, manufacturers, cars (using created drivers). Try to remove driver from car too. Don't access them by hardcoded IDs, use getId() after saving in DB